cmd/snap/cmd_prepare_image.go

```diff
@@ -120,13 +123,17 @@ func (x *cmdPrepareImage) Execute(args []string) error {
 	// level for "snap" command, for seed/seedwriter used by image however
 	// we want real validation.
 	snap.SanitizePlugsSlots = builtin.SanitizePlugsSlots
+	imageCustomizations := image.Customizations{
```

Comment:
would `--validation` passed through the command override what is provided in customizations JSON file? I think it should, then this should be moved right after `x.Customize != ""` check

Reply:
Also, it'd be nice to add a short unit test for completeness

Reply:
Uh, I totally missed that. Well spotted, than you!
Tests now added as well

Reply:
@bboozzoo just for context customization is a hidden option because it was meant only for the Python version of ubuntu-image that was calling prepare-image as a command, so we don't expect it to get mixed with other arguments, it doesn't hurt to support it though